README.md

```diff
@@ -147,18 +147,37 @@ Thank you to the following individuals for contributing to this release:
 * Mauricio Vásquez
 * Yusuke Tsutsumi
 
+The [v0.5 beta
+release](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v0.5.0) release includes:
+
+- W3C Correlation Context Propagation
+- OpenTelemetry Collector Exporter Integration for both metrics and traces
+- Metrics SDK
+- Global configuration module
+- Documentation improvements
+
+Thank you to the following individuals for contributing to this release:
+
+* Alex Boten
+* Chris Kleinknecht
+* Dave Grochowski
+* Diego Hurtado
+* Hector Hernandez
+* Leighton Chen
+* Liz Fong-Jones
+* Mauricio Vásquez
+* Yusuke Tsutsumi
+
 See the [project
 milestones](https://github.com/open-telemetry/opentelemetry-python/milestones)
 for details on upcoming releases. The dates and features described here are
 estimates, and subject to change.
 
 Future releases targets include:
 
-| Component                           | Version    | Target Date  |
-| ----------------------------------- | ---------- | ------------ |
-| W3C Correlation Context Propagation | Beta v1    | March 16 2020|
-| Support for Tags/Baggage            | Beta v1    | March 16 2020|
-| gRPC Integrations                   | Beta v1    | March 16 2020|
-| OpenTelemetry Collector Exporter    | Beta v1    | March 16 2020|
-| OpenCensus Bridge                   | Beta v1    | March 16 2020|
-| Metrics SDK (Complete)              | Beta v1    | March 16 2020|
+| Component                          | Version | Target Date   |
+| ---------------------------------- | ------- | ------------- |
+
+| Stable API for metrics and tracing | Beta v2 | March 31 2020 |
```

Comment:
Are users aware of what is "Beta v2"?, could "v0.6.0" be better?

Reply:
I agree this may be better, but "beta" is at least consistent with the other clients and the language in the launch announcement.